src/services/product/product.service.ts

```diff
@@ -50,10 +55,15 @@ export class ProductService {
     );
   }
 
-  async getAllProducts(paginationDto: PaginationDto) {
-    const { take = 10, skip = 0 } = paginationDto;
+  async getAllProducts(getProductsDto: GetProductsDto) {
+    const {
+      take = 10,
+      skip = 0,
+      sort = SortProductsCriteria.CREATED_AT,
+      order = OrderCriteria.ASC
+    } = getProductsDto;
 
-    return await this.productRepository.getAll(take, skip);
+    return await this.productRepository.getAll(take, skip, sort, order);
```

Comment:
Hey Bruno! here the await it's not necessary,  based on our conversation on slack. Here, the function that is using `getAllProducts` implements the await so, here it's not necessary